src/Smalot/PdfParser/Font.php

```diff
@@ -381,86 +384,151 @@ public function decodeText(array $commands): string
     public function decodeContent(string $text, ?bool &$unicode = null): string
     {
         if ($this->has('ToUnicode')) {
-            $bytes = $this->tableSizes['from'];
+            return $this->decodeContentToUnicode($text);
+        }
 
-            if ($bytes) {
-                $result = '';
-                $length = \strlen($text);
+        if ($this->has('Encoding')) {
```

Comment:
Please use if-elseif here.

Reply:
I think if-return variant has better readability, hasn't it?

I googled a bit and found that some developer prefer if-elseif to have just one point of return statement. But I think it is not a good reason for this case because there is no any common code for some if-branches or all-branches common post-processing.

 Also I also looked through how elseif already used in project and not found where it used with return operator.

Also there is already exists using if-return some times:
- [Document::getObjectsByType()](https://github.com/smalot/pdfparser/blob/master/src/Smalot/PdfParser/Document.php#L188) and [Docuemnt::getPages()](https://github.com/smalot/pdfparser/blob/master/src/Smalot/PdfParser/Document.php#L228);
- [Page::getForns()](https://github.com/smalot/pdfparser/blob/master/src/Smalot/PdfParser/Page.php#L58);

Therefore I think it would be better to keep if-return statements as is.

Do you agree with me?

Reply:
Good points, I agree.